environment.yml

```diff
@@ -1,38 +1,29 @@
-name: xcube
+name: xcube5
 channels:
   - conda-forge
-  - defaults
 dependencies:
   # Python
-  - python >=3.6
-  # Required
+  - python
```

Comment:
Why remove `# Required` comment, why don't we need a minimum Python?

Reply:
Only change now is xarray version